### ISSUE TITLE
[BUG] Better error message for 'backward'

### ISSUE THREAD

**Issue by MarleenGilsing** [Felienne/hedy#1937]
At the moment Hedy gives the 'you forgot to use a command' error when using backward in the turtle. While I think that would be a very logical command for a student to try. So maybe create an error message that tells you backward isn't a command but you can use forward and negative numbers. 

![image](https://user-images.githubusercontent.com/80678586/153416983-80623aae-80e7-40bb-a31f-a96536cde7fc.png)

**Comment by huang-v** [comment]
Would a custom error message for the first level be desired?

**Comment by MarleenGilsing** [comment]
@huang-v Yes it is, thanks! I would like the error message to make clear that the turtle can go backwards but they have to use forward -100.

**Comment by noelgolding** [comment]
I would argue this is not a bug, rather an enhancement request.

**Comment by noelgolding** [comment]
curious, not sure why my last comment was deleted.  I was simply suggesting this issue may be better labeled as an enhancement rather than a bug.

**Comment by Felienne** [comment]
It was removed because it is not helpful. Feel free to drop in on Discord
or in a meeting, drive by comments just add to our workload and solve
nothing.

**Comment by noelgolding** [comment]
Ah, ok. I thought it was being helpful to reorganize the issue for prioritization. I will join the discord, thank you. On Jan 2, 2023, at 4:22 PM, Felienne Hermans ***@***.***> wrote:﻿
It was removed because it is not helpful. Feel free to drop in on Discord
or in a meeting, drive by comments just add to our workload and solve
nothing.
